lib/sprangular/engine.rb

```diff
@@ -13,6 +13,12 @@ class Engine < ::Rails::Engine
       Rails.application.config.assets.precompile += %w( images/* bootstrap/* )
     end
 
+    initializer "sprangular.api_domain", :before => :load_config_initializers do |app|
+      Spree::AppConfiguration.class_eval do
+        preference :api_domain, :string, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.